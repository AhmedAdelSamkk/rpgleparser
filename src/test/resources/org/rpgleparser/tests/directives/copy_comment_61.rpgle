     D/COPY QASWCPYPRC,AC00001                   ASWJob  
