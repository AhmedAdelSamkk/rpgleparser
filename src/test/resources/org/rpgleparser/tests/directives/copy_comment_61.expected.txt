/*===TOKENS===*/
HIDDEN:DIRECTIVE         <D/>
HIDDEN:DIR_COPY          <COPY>
HIDDEN:DIR_OtherText     <QASWCPYPRC>
HIDDEN:DIR_Comma         <,>
HIDDEN:DIR_OtherText     <AC00001>
/*===TREE===*/
r
/*======*/